samples/subsys/usb/hid-mouse/src/main.c

```diff
@@ -153,7 +153,7 @@ void main(void)
 		k_sem_take(&sem, K_FOREVER);
 
 		report[0] = status;
-		usb_write(CONFIG_HID_INT_EP_ADDR, report,
+		usb_write(CONFIG_HID_INT_IN_EP_ADDR, report,
```

Comment:
It is not very clean,  these sample are broken between the commits. It would be better to make a patch renaming all CONFIG_HID_INT_EP_ADDR in CONFIG_HID_INT_IN_EP_ADDR. Then a patch that introduces CONFIG_HID_INT_OUT_EP_ADDR.